src/CMake/glibc.cmake

```diff
@@ -12,7 +12,12 @@ if (DEFINED CROSS_COMPILE)
 endif()
 get_filename_component(GLIBC ${GLIBC} REALPATH)
 get_filename_component(GLIBC_VERSION ${GLIBC} NAME)
-string(REPLACE "libc-" "" GLIBC_VERSION ${GLIBC_VERSION})
+if ((${LINUX_FLAVOR} STREQUAL "Ubuntu") 
```

Comment:
Besides https://cmake.org/cmake/help/latest/command/if.html#version-greater-equal I do not like this super ad-hoc solution.
Ubuntu is just one version of Debian, not event talking about other Linux distribution families.
Could you come with a generic solution instead?
Going back to https://gist.github.com/likema/f5c04dad837d2f5068ae7a8860c180e7 you can come with a solution working for both `libc` and `libc-` independently from the Linux distribution?

Reply:
sure, thanks for the suggestion

Reply:
Hi @keryell , this file is not needed anymore as dependencies are auto added since pr - https://github.com/Xilinx/XRT/pull/3809

Reply:
I love when these kinds of atrocities just disappear naturally. :-) Great!